src/pdf/DayOverview/DayOverviewPdfButton.tsx

```diff
@@ -64,9 +66,29 @@ export interface DayOverviewPdfData {
     parcels: ParcelForDayOverview[];
 }
 
-type DayOverviewPdfErrorType = ParcelForDayOverviewErrorType;
+type DayOverviewPdfErrorType = ParcelForDayOverviewErrorType | CongestionChargeError;
 export type DayOverviewPdfError = { type: DayOverviewPdfErrorType; logId: string };
 
+const addCongestionChargeDetailsForDayOverview = async (
+    parcels: ParcelForDayOverview[]
+): Promise<ParcelForDayOverviewResponse> => {
+    const postcodes = parcels.map((parcel) => parcel.client?.address_postcode || null);
+
```

Comment:
just noticed I think it should be ?? not ||
can you make sure the resulting array is as expected?

Reply:
The array is as expected - in this context I believe the two work in the same way?

Reply:
(With a slight difference if the postcode was an empty string)

Reply:
Yeah i agree, happy to leave it however you want, I think I would use ?? but i don't think it matters much
happy for you to ask for reviews from Ricky/Kristina now

Reply:
Thanks!